specification/quota/Quota.Management/CurrentQuotaLimitBase.tsp

```diff
@@ -23,6 +23,14 @@ model CurrentQuotaLimitBase
   >;
 }
 
+alias CurrentQuotaLimitBasesResponseHeaders = {
+  /**
+   * Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.
+   */
+  @header("ETag")
+  eTag: string;
```

Comment:
Do you think we can use `Legacy.EntityTagProperty` (this is newly added) to mitgate `etag` change? 
example: https://github.com/Azure/azure-rest-api-specs/pull/36066/commits/680bb1d07ecc9e605984037f0194e5f0a619bf72

Reply:
This one is not a header?

Reply:
However, we may have a bug in emitter, that clientName is not handled in header/query. Need to check there.

Reply:
Yeah, on the authoring part, ARM typically uses etag in model. It is particularly strange to return etag header on a list, as etag generally used to version a single resource, not a list of them.

But we'd just let service write whatever they want...